FrontEnd/src/components/AboutPage/MeetTheTeam/TeamMemberInfo.tsx

```diff
@@ -0,0 +1,102 @@
+const memberCards = [
```

Comment:
This file can be a .ts file instead of a .tsx file. .ts represents typescript files and .tsx represents typescript files that also use JSX (aka react). We want to be as "strict" as possible, and there's no need for a .tsx extension here so let's swap it to .ts